regen-network/regen-web
# fix: APP-135 update Save button disabled functionality and styles

## Description

Changes:
- Ensure Save button styles are the same in both the Project Edit and Edit Profile pages
- Refactor the way we watch Edit Profile form fields to track form state and implement Save button disabled state.

---

### Author Checklist

I have...


### How to test

1. Log in into the Marketplace and verify that the Save button in both the Edit Profile and Project Edit page have the same styles. 
2. Update the Edit Profile form to ensure the Save button is enabled when any field changes and disabled when no changes are made.

Marketplace: https://deploy-preview-2396--regen-marketplace.netlify.app/profile/edit/profile

### Reviewers Checklist

_All items are required. Please add a note if the item is not applicable and please **add
your handle next to the items reviewed if you only reviewed selected items**._

I have...
